sources/settings-plugins/aws-k8s-nvidia/Cargo.toml

```
@@ -0,0 +1,29 @@
+[package]
+name = "settings-plugin-aws-k8s-nvidia"
+version = "0.1.0"
+edition = "2021"
+license = "Apache-2.0 OR MIT"
+publish = false
+
+[lib]
+crate-type = ["cdylib"]
+name = "settings_aws_k8s_nvidia"
+
+[dependencies]
+abi_stable = "0.11.3"
+serde = "1.0.198"
+serde_json = "1.0.116"
+
+# settings plugins
+[dependencies.bottlerocket-settings-plugin]
+# ======= REMOVE BEFORE FINAL PR =================
+git = "https://github.com/bottlerocket-os/bottlerocket-settings-sdk"
+rev = "4709409481ebc0c2f90bb993c257cd9d670190bf"
+version = "0.1.0"
+
+[dependencies.bottlerocket-settings-models]
+# ======= REMOVE BEFORE FINAL PR =================
+git = "https://github.com/bottlerocket-os/bottlerocket-settings-sdk"
+rev = "4709409481ebc0c2f90bb993c257cd9d670190bf"
+version = "0.2.0"
+features = ["nvidia-device-plugin"]
```

Why define these as workspace dependencies if we're not going to use them?
```suggestion
bottlerocket-settings-plugin = { workspace = true }
bottlerocket-settings-models = { workspace = true, features = ["nvidia-device-plugin"] }
```

---

changed as suggested.